Foxtrot/Tests/ExtractorTest.cs

```diff
@@ -12,47 +12,262 @@
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1015.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: @"/optimize",
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1016.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CheckerErrors.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+            }
+        }
+
+        public static IEnumerable<object> ExtractorTest
+        {
+            get
             {
-                try
-                {
-                    string sourceFile = (string)TestContext.DataRow["Name"];
-                    string options = (string)TestContext.DataRow["Options"];
-                    TestContext.WriteLine("FAILED: Name={0} Options={1}", sourceFile, options);
-                }
-                catch { }
+                return Enumerable.Range(0, ExtractorTestData.Count()).Select(i => new object[] { i });
```

Comment:
The problem is the way we are using the parameters. `Options` is a mutable class which provides both behavior and data, but the data structures used to parameterize tests *should* be immutable and only contain the test data. This would be a problem regardless of the test framework in use. This is something we can improve on in the future, but for now I'm just happy to see all the tests able to run on AppVeyor and start getting some reports. :smile: